modules/network-observability-operator-install.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-operator-installation_{context}"]
+
+= Installing the Network Observability operator
+You can install the Network Observability operator using the {product-title} web console Operator Hub.
+ 
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+//In the Operator Hub on the test console Julian provided perms for, its NetObserv Operator, with a capital O. 
+.. Choose  *Network Observability operator* from the list of available Operators in the *OperatorHub*, and click *Install*.
+.. Navigate to *Operators* -> *Installed Operators*. Under Provided APIs for Network Observability, slect the *Flow Collector* resource.
+. Navigate to the *Flow Collector* tab, and click *Create FlowCollector*. Make the following slections in the form view:
++
+- *Sampling* : Specify a sampling size that will not consume too many resources. For more information, see the FlowCollector custom resource configuration parameters reference table. 
+- *deploymentModel*: If you are using Kafka, make sure Kafka is selected.
+- *Loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://lokistack.gateway.http.netobserv.svc.8080/api/logs/v1/network`
+-*authToken*: Select `FORWARD`
```

Comment:
Yes, I would opt for HOST for simplicity.
FORWARD is for multi-tenancy which is only half-baked at the moment, it will come with 4.13, but it's simpler to not mention it for now

Reply:
Addressed